modal/_container_entrypoint.py

```diff
@@ -30,6 +31,7 @@
 from ._container_io_manager import ContainerIOManager, UserException, _ContainerIOManager
 from ._proxy_tunnel import proxy_tunnel
 from ._serialization import deserialize
+from ._telemetry import instrument_imports
```

Comment:
I mean that you move this import inside the `if` statement below, to avoid loading the telemetry module for all users  👇

Reply:
Yeah that's what I thought as well! Now actually done 😄